net/unbound/files/root.hints

```diff
@@ -0,0 +1,39 @@
+.			518400	IN	NS	a.root-servers.net.
```

Comment:
I thought so, but then had a different experience. The version booked for CC 15.05.01 only appears to update root.key, but not root.hints/named.cache. The built ins can become outdated. Please confirm.